install/nightlies.md

```diff
@@ -0,0 +1,62 @@
+---
+subtitle: Nightly Builds
+---
+
+Nightly builds of the Crystal compiler are available from these locations:
+
+## Tarball
+
+Tarballs with nightly builds are available for the following platforms:
+
+* linux x86_64: [`https://artifacts.crystal-lang.org/dist/crystal-nightly-linux-x86_64.tar.gz`](https://artifacts.crystal-lang.org/dist/crystal-nightly-linux-x86_64.tar.gz)
+* darwin universal (x86_64 and aarch64): [`https://artifacts.crystal-lang.org/dist/crystal-nightly-darwin-universal.tar.gz`](https://artifacts.crystal-lang.org/dist/crystal-nightly-darwin-universal.tar.gz)
+* windows x86_64: [`https://nightly.link/crystal-lang/crystal/workflows/win/master/crystal.zip`](https://nightly.link/crystal-lang/crystal/workflows/win/master/crystal.zip)
+
+See [*Install from tar.gz*](/install/from_targz) for further instructions.
```

Comment:
Any reason for the italics here? (more a curiosity than a concern)
```suggestion
See [Install from tar.gz](/install/from_targz) for further instructions.
```

Reply:
Cause it's a page title 🤷